opendebates/admin.py

```diff
@@ -103,3 +104,8 @@ class SiteModeAdmin(ModelAdmin):
     _missed_fields = set(_fields) - _used_fields
     if _missed_fields:
         fieldsets.append(('Other', {'fields': sorted(list(_missed_fields))}))
+
+    def reset_current_votes(self, request, queryset):
+        # Note, currently all submissions are under the one and only SiteMode.
+        models.Submission.objects.update(current_votes=0)
```

Comment:
I like the explicitness too, but, actually, what is the sequence that moderators would have to follow here?  I think things will have to be done in a specific order to keep the tallies accurate:

1. Set previous_debate_time to something in the future
2. Run the admin action
3. Wait for time to pass

If 1<->2 are flipped, or if you set previous_debate_time to something that isn't yet in the future, ISTM we'd end up with tallies that aren't quite accurate.

If this is correct, then I think it also supports my suggestion that the sort option should only become visible if previous_debate_time is in the past - otherwise there'd be no way to take the necessary steps without turning on the UI prematurely.